packages/appcd-core/src/server.js

```diff
@@ -162,6 +162,15 @@ export default class Server {
 		await this.systems.telemetry.init(homeDir);
 		Dispatcher.register('/appcd/telemetry', this.systems.telemetry);
 
+		// report crash telemetry
+		process.on('uncaughtException', async err => {
+			await Dispatcher.call('/appcd/telemetry', {
+				event: 'crash.report',
+				message: err.message,
+				stack: [ err.stack ]
```

Comment:
`err.stack` is a string. Why wrap it in an array instead of just doing `err.stack.split(/\r\n|\n/)`?

Reply:
This is the format of crash events used in other projects (as you can supply multiple stack traces).